src/app/(marketing)/profile/_components/EditProfile.tsx

```diff
@@ -0,0 +1,216 @@
+                        </Button>
+                      </div>
+                    </FormItem>
+                  )}
+                />
+                <FormField
+                  control={form.control}
+                  name="profileImageUrl"
+                  render={({ field }) => (
+                    <Button
+                      variant="unstyled"
+                      type="button"
+                      className={cn(
+                        "-bottom-2 absolute left-4 size-28 translate-y-1/2 rounded-full border-2 border-dotted",
+                        { "border-solid": field.value }
+                      )}
+                      {...getRootProps()}
+                    >
+                      <Avatar className="size-36 bg-card-secondary">
+                        {field.value && (
+                          <AvatarImage
+                            src={field.value}
+                            width={144}
+                            height={144}
+                            alt="profile avatar"
+                          />
+                        )}
+                        <AvatarFallback className="bg-card-secondary">
+                          <User size={38} />
+                        </AvatarFallback>
+                      </Avatar>
+                    </Button>
+                  )}
+                />
+              </div>
+
+              <FormField
+                control={form.control}
+                name="name"
+                render={({ field }) => (
+                  <FormItem className="pb-2">
+                    <FormLabel>Name</FormLabel>
+                    <FormControl>
+                      <Input placeholder="" {...field} />
+                    </FormControl>
+                    <FormErrorMessage />
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="username"
+                render={({ field }) => (
+                  <FormItem className="pb-2">
+                    <FormLabel aria-required="true">Username</FormLabel>
+                    <FormControl>
+                      <Input placeholder="" required {...field} />
+                    </FormControl>
+                    <FormErrorMessage />
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="bio"
+                render={({ field }) => (
+                  <FormItem className="">
+                    <FormLabel>Bio</FormLabel>
+                    <FormControl>
+                      <Textarea placeholder="" className="max-h-12" {...field} />
+                    </FormControl>
+                    <FormErrorMessage />
+                  </FormItem>
+                )}
+              />
+            </DialogBody>
+            <DialogFooter>
+              <Button colorScheme="destructive" type="submit">
```

Comment:
This button is too strong in my opinion, I think we should try to use another variant. Maybe outline? And I'm not 100% sure if we actually need this functionality, because in theory if the user cancels their subscription, we just delete their profile, right? (Except for the lifetime subscription, I think)

Reply:
This button is actually temporary, and only needed for development.